src/modules.json

```diff
@@ -361,6 +361,11 @@
     },
     "util": {
       "js_file": "js/util.js"
+    },
+    "bridge": {
```

Comment:
Please remove this change.

Reply:
Ok, I will add README.md for sample_module, but I think src/modules.json is right place for the bridge module..